sql/src/test/java/org/apache/druid/sql/calcite/CalciteWindowQueryTest.java

```diff
@@ -339,6 +327,9 @@ enum TestType
     @JsonProperty
     public TestType type;
 
+    @JsonProperty
+    public Map<String, String> queryContext;
```

Comment:
Nice to have query context extended here providing the flexibility removing `windowQueryTestWithCustomContextMaxSubqueryBytes`!